docs/troubleshooting/semgrep.md

```diff
@@ -7,62 +7,8 @@ import MoreHelp from "/src/components/MoreHelp"
 
 # Troubleshooting Semgrep
 
-## Troubleshooting Semgrep CI/action/agent
```

Comment:
Just so you know... @emjin suggested deleting this section as the whole troubleshooting is now different probably. Let me know if we rather should keep this still - either only for Semgrep CI, rewrite these sections a bit or mark it: Troubleshooting legacy Semgrep action/agent (because it is still legacy for some users, right? They are going to use it still.) @underyx @brendongo

Reply:
I'd just update `SEMGREP_REPO_NAME=<your-org-here>/<repo-name-here> semgrep --config policy` to `SEMGREP_REPO_NAME=<your-org-here>/<repo-name-here> semgrep ci`

Reply:
Ok! Changed!